src/app/components/RuntimeEvents/RuntimeEventDetails.tsx

```diff
@@ -218,10 +219,11 @@ export const RuntimeEventDetails: FC<{
           <StyledDescriptionList titleWidth={isMobile ? '100px' : '200px'}>
             <dt>{t('runtimeEvent.fields.owner')}</dt>
             <dd>
-              <AccountLink address={event.body.owner} scope={scope} />
-              {addressSwitchOption === AddressSwitchOption.Oasis && (
-                <CopyToClipboard value={event.body.owner} />
-              )}
```

Comment:
This `CopyToClipboard` was implemented with combination of plain mode on the address. Which is now gone, so unsure if we even need it anymore? Or should it be shown for both type of addresses.

Reply:
it is useful in mobile. not sure why it  was limited to "oasis addr"

Reply:
This was due to, "plain" mode in AccountLink component for oasis type addresses.

The current implementation made no sense for me, so if we should include it, I think it should be for both type of addresses or remove it.